src/JsonField.tsx

```diff
@@ -15,7 +16,7 @@ export const JsonField: React.FC<Props> = ({
 }) => {
   const record = useRecordContext();
```

Comment:
all components use a pattern

```
const record = useRecordContext(props);
```

this allows passing `record={{override: ...}}` with `<FunctionField>`  this enabled pretty much any type of transformation

`get(record, source)` doesn't compose with `jsonString` being done a line below

anyway this seems like a dead repo :/